### DISCUSSION TITLE
New extension python stubs

### DISCUSSION THREAD

**Discussion by Real-Gecko** [microsoft/pylance-release#2801] [Q&A]
I'm creating a new VSCode extension and I want to add some `*.pyi` files for type hinting and autocompletion. How do I do that?

**Comment by brettcannon** [comment]
Are you asking how to write `.pyi` files or how to have Pylance pick them up?

**Comment by Real-Gecko** [comment]
How do I make Pylance find them?

**Reply by brettcannon** [reply]
I believe `python.analysis.stubPath` is the setting you're looking for.

**Comment by Real-Gecko** [comment]
Do I set in my `settings.json`? Is there a way to set in my extension `package.json`?

**Reply by brettcannon** [reply]
It's for `settings.json`. I don't remember off the top of my head, but there is a way to have extensions specify settings for the user.
